loadouts/factions/structures/afrenian_army.sqf

```diff
@@ -73,8 +73,7 @@ switch (true) do {
     case (_isTeamLeader): {
         [_unit, _leaderHEAD, _commonUNIFORM, _ftlVEST, "empty"] call BRM_FMK_fnc_useUniform;
         [_unit,[[_wsmoke,2],[_rsmoke,2],[_flashlight,1]]] call BRM_FMK_fnc_addtoVest;
-        [_unit, _commonRIFLEGL, _countRIFLE] call BRM_FMK_fnc_addWeaponKit;
```

Comment:
This should be fixed in the [loadout](https://github.com/Neefay/BromA-Framework-A3-Addon/blob/master/loadouts/factions/afrenian.sqf#L28), not the structure.

Reply:
Afrenia army loadout use standalone GL, not rifle with GL, so the TL must have it's structure changed so he use a rifle, if you look at grenadier you'll see that he has both the rifle AND the standalone GL, also this structure is only for afrenia, not for everyone ( unless someone decide so )